### ISSUE TITLE
FIREFOX Stackblitz Embeds in docs give "Unable to run Embedded Project" error on the preview pane

### ISSUE THREAD

**Issue by GeorgeWL** [TanStack/tanstack.com#217]
the specific details of the error are:
> Unable to run Embedded Project
>
> Looks like this project is being embedded without proper isolation headers.
> [Read more about browser support for embedded projects](https://developer.stackblitz.com/platform/webcontainers/browser-support#embedding)

**Comment by SeanCassiere** [comment]
I don't think we can really do anything about this, since this is from Stackblitz, which is our preferred sandbox envs.

**Comment by GeorgeWL** [comment]
You could raise it with them. They have a repo for issues
